tests/execute/result/repeated.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
```

Comment:
Hm, `beakerlib` is not installed. So for these we probably do some kind of ignore? @jamacku, any thoughts?

Reply:
The best solution would be to install beakerlib. I have opened PR for it:
- https://github.com/teemtee/tmt/pull/1967

Reply:
> Small note: If you decide the defect is not relevant and you will merge this PR, it won't get reported again, even without the ShellCheck directive.

This would apply for the unassigned `$run` and `$rlRun_LOG` variables as well?